tests/posix/threads_base/src/pthread.c

```diff
@@ -469,14 +466,14 @@ ZTEST(pthread, test_sched_getparam)
 	zassert_true((rc == -1 && err == ENOSYS));
 }
 
-ZTEST(pthread, test_sched_getscheduler)
+ZTEST(posix_threads_base, test_sched_getscheduler)
 {
 	int rc = sched_getscheduler(0);
 	int err = errno;
 
 	zassert_true((rc == -1 && err == ENOSYS));
 }
-ZTEST(pthread, test_sched_setparam)
+ZTEST(posix_threads_base, test_sched_setparam)
```

Comment:
This test belongs in the `xsi_realtime` testsuite with the `_POSIX_PRIORITY_SCHEDULING` Option.